lib/expressroutes.js

```diff
@@ -173,18 +179,18 @@ function authorizeFor(security) {
                         return;
                     }
                     pass(true);
-                });
+                }, securityDefinition);
```

Comment:
Oh wait, yeah we can't pass this as the last argument. Bad form to pass data after a callback.